packages/GoogleAnalytics/src/components/WithGATracker/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component, ComponentType } from 'react';
+import { ConnectedComponent } from 'react-redux';
+import { RouteComponentProps } from 'react-router';
+import { trackPage } from '../../helpers';
+
+type Props = RouteComponentProps;
+type FlexComponent<T = {}> = ComponentType<T>;
```

Comment:
Do you think that we should move this to [@onaio/utils](https://github.com/onaio/js-tools/tree/master/packages/utils)?